### PR TITLE
changed from group.name to group.long_name in serializer

### PR DESCRIPTION
**Pull request by s0lvang**
Fixing dump data continued

### CODE REVIEW

**Review by FluidSense — APPROVED** [review]
Just try 👌

File: apps/authentication/api/serializers/user_data.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ class Meta:
 
 class WikiArticleSerializer(serializers.ModelSerializer):
     current_revision = WikiArticleRevisionSerializer()
-    group = serializers.CharField(source="group.name")
+    group = serializers.CharField(source="group.name_long")
```
Comment (oleast) [inline apps/authentication/api/serializers/user_data.py:143]: I think this group is a Django group, which doesn't have a long name?

Comment (FluidSense) [reply apps/authentication/api/serializers/user_data.py:143]: The error says that it doesn't have a name either, which is why my comment is "Just try" 😅 😂 Neither is correct, any idea of what it should be @oleast ?

Comment (oleast) [reply apps/authentication/api/serializers/user_data.py:143]: It's supposed to be group :/

https://github.com/django-wiki/django-wiki/blob/master/src/wiki/models/article.py#L68

Comment (FluidSense) [reply apps/authentication/api/serializers/user_data.py:143]: Hm, seems like the actual issue is that it is a nullable field, which then crashes on "None.name" when there is no group.
https://www.django-rest-framework.org/api-guide/fields/
Perhaps one could pass the parameter "default" with argument "Ingen gruppe", to handle such cases.

**Pull request merged**